scene/resources/audio_stream_wav.cpp

```diff
@@ -28,11 +28,19 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-#include "audio_stream_wav.h"
-
-#include "core/io/file_access_memory.h"
 #include "core/io/marshalls.h"
 
+#define DRWAV_IMPLEMENTATION
+#define DR_WAV_NO_STDIO
+#define DR_WAV_LIBSNDFILE_COMPAT
+#define DRWAV_MALLOC(sz) memalloc(sz)
+#define DRWAV_REALLOC(p, sz) memrealloc(p, sz)
+#define DRWAV_FREE(p) \
+	if (p)            \
+	memfree(p)
```

Comment:
I'm not sure to have this here. Wouldn't it be cleaner if we put this into a .cpp file inside thrdparty/dr_libs?

Reply:
Done

Reply:
I think I wasn't clear enough of what I had in mind! :sweat_smile: 

Either move this to an header file that you import afterwards (like `dr_wav_godot.h`) or implement both `dr_wav_godot.h` and `dr_wav_godot.cpp` that acts like a wrapper for dr wav.

Currently, I don't think your new cpp file does anything.

Reply:
I moved the defines to a file named `dr_wav_defines.h` and included it in `audio_stream_wav.cpp`